hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/RecordStore.java

```diff
@@ -409,4 +412,6 @@
     boolean isKeyLoadFinished();
 
     InvalidationQueue<ExpiredKey> getExpiredKeys();
+
+    boolean hasQueryCache();
```

Comment:
Minor: you can add the javadoc here as well

Reply:
done